README.md

```diff
@@ -1,21 +1,36 @@
-# metrics [![][travis_img]][travis_url] [![][npm_img]][npm_url]
+# @mongodb-js/compass-metrics [![][travis_img]][travis_url] [![][npm_img]][npm_url]
 
-> Compass Metrics Plugin
+A [MongoDB Compass](https://github.com/mongodb-js/compass) plugin that allows you to help the team by [submitting error diagnostics and telemetry](https://docs.mongodb.com/compass/master/faq/#how-do-i-view-and-modify-my-privacy-settings). it configures [the mongodb-js-metrics module](https://github.com/mongodb-js/metrics) to handle data transport, along with a manifest of what information could be collected (see [./src/modules/rules.js](src/modules/rules.js)).
+
+The metrics plugin is heavily inspired by [the Atom metrics package](https://atom.io/packages/metrics).
 
 ## Usage
 
-## License
+To open the Import File dialog, a plugin emits `open-import`:
 
-Apache 2
+```
+dispatch(globalAppRegistryEmit('open-import'));
+```
 
-===
+[./src/modules/rules.js](src/modules/rules.js) defines what events in Compass we subscribe to for telemetry and the associated metadata we'll capture. So we can answer our question with this small rule:
```

Comment:
non-blocking comment: would be good to have stuff in `Usage` about what gets appended to all events vs. what individual event metadata gets registered (things like `created_at` or `compass_version` are for example all appended to each event)

Might be good to also give an example of an event that receives arguments too?